scripts/publish-app.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+var ghpages = require('gh-pages');
+var path = require('path');
+var fs = require('fs');
+var args = require('yargs')
+  .boolean('p')
+  .default('p', false)
+  .alias('p', 'push')
+  .describe('p', 'Push changes to remote repository, i.e. do an actual publish')
+  .alias('v', 'version')
+  .describe('v', 'semver version to publish the app as')
+  .demandOption(['p'])
+  .demandCommand(1)
+  .usage('Usage: $0 <app name> -p ')
+  .argv;
+
+// Debug tooling
+process.env.NODE_DEBUG = 'gh-pages';
+
+// get version from lerna global version
+var version = args.version || require(path.resolve(__dirname, '../lerna.json')).version;
+var appName = args._[0];
+
+var appInfo = {
+  server: {
+    path: 'demo/server',
+    repo: 'git@github.com:feedhenry-templates/wfm-server.git'
+  },
+  mobile: {
+    path: 'demo/mobile',
+    repo: 'git@github.com:feedhenry-templates/wfm-portal.git'
+  },
+  portal: {
+    path: 'demo/portal',
+    repo: 'git@github.com:feedhenry-templates/wfm-mobile.git'
+  },
+};
+
+var target = appInfo[appName];
+
+if(!target || !fs.existsSync(target.path)) {
+  console.error('module ' + appName + ' not found!');
+}
+
+var options = {
+  branch: version,
+  dotfiles: true,
+  //Only add, and never remove existing files
+  add: true,
+  remote: 'origin',
+  repo: target.repo,
+  user: { name: "Wojciech Trocki", email: "wtrocki@redhat.com" },
```

Comment:
I put my names to check if github user will be mapped (it works).
We may allow to override that? I'm not sure as mostly releases will be done by single person.